### PR TITLE
Commented out blog section on home page

### PR DESCRIPTION
**Pull request by Chrissle28**
- I commented out the blog section on the home page
- The tag on a blog post will only be visible if there is an image
- the active switch will now work (only showing blog elements with active activated)

### CODE REVIEW

File: resources/views/pages/home.blade.php [hunk 1]
```diff
@@ -112,7 +112,8 @@
 
 
 
-{{-- <section class="py-8 bg-white md:py-20 lg:py-40">
+@if(count($posts) > 0)
```
Comment (Chrissle28) [inline resources/views/pages/home.blade.php:115]: @lpheller like so?

Comment (lpheller) [reply resources/views/pages/home.blade.php:115]: Yep, exactly. Probably could do it like this as well:
```suggestion
@if(!$posts->isEmpty())
// or
@if($posts->isNotEmpty()
```
but that shouldn't matter, imo it just reads a little nicer.

Comment (lpheller) [reply resources/views/pages/home.blade.php:115]: Or
```suggestion
@if($posts->isNotEmpty())
```

**Review by lpheller — APPROVED** [review]

File: resources/views/pages/home.blade.php (hunk 1 above)
Comment (Chrissle28) [reply resources/views/pages/home.blade.php]: I added the ->isNotEMpty() method

**Pull request merged**